# Header for timestamp and frame_id
std_msgs/Header header

# 3D coordinates of the four outer corners of the chessboard
geometry_msgs/Point top_left
geometry_msgs/Point top_right
geometry_msgs/Point bottom_left
geometry_msgs/Point bottom_right